addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightHandlerBase.java

```diff
@@ -151,6 +179,10 @@ void handleColorTemperatureChannelCommand(ChannelUID channelUID, Command command
         }
     }
 
+    void handleRefreshCommand() {
+        DeviceManager.getInstance().startDiscovery(15 * 1000);
```

Comment:
Is that the right action to do on a refresh or is something else expected?

Reply:
A refresh should initiate a state update for the channel for which it is sent.

Reply:
I updated the function now. I guess a state update tries to fetch the current values of the device for the channel? I just had a quick look, but I did not find such methods in the lib. So I just added ad TODO and the construct.

Reply:
The lib seems to offer such methods in DeviceStatus probably, so they can be implemented.